### PR TITLE
AC Test Fix - LRAC-10065 | master 

### PR DESCRIPTION
**Pull request by manuferreira**
https://testray.liferay.com/home/-/testray/subtasks/1588438343
https://testray.liferay.com/home/-/testray/subtasks/1588939582
https://issues.liferay.com/browse/LRAC-10065 (ticket)
https://issues.liferay.com/browse/LRAC-10049 (bug)

### CODE REVIEW

**Comment by manuferreira** [comment]
ci:test:sf

**Comment by manuferreira** [comment]
ci:test:sf

**Comment by manuferreira** [comment]
ci:test:sf

**Comment by manuferreira** [comment]
ci:test:sf

**Review by ViniLopes20 — APPROVED** [review]

**Comment by manuferreira** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationWebBehavior.testcase [hunk 1]
```diff
@@ -1243,8 +1270,12 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-9226 | Test Summary: Create a Web Behavior Segment of Viewing a Web Content"
+	@ignore = "true"
 	@priority = "4"
 	test CanCreateWebBehaviorSegmentViewingWC {
+
+		// AC Bug Ticket: 10049
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationWebBehavior.testcase:1277]: hi @manuferreira , the `LRAC-`10049 is missing. ;)

**Comment by alexcandidos** [comment]
Hi @manuferreira , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**